Dockerfile

```diff
@@ -92,6 +96,11 @@ ADD amazonhiveodbc_2.6.9.1009-2_amd64.deb /opt/dataworks/hiveodbc.deb
 RUN dpkg -i /opt/dataworks/hiveodbc.deb \
     && rm -rf /opt/dataworks/hiveodbc.deb
 
+RUN cd /opt/dataworks && wget -q https://dlcdn.apache.org/hadoop/common/hadoop-3.3.1/hadoop-3.3.1-aarch64.tar.gz && \
+    wget -q https://dlcdn.apache.org/hive/hive-3.1.2/apache-hive-3.1.2-bin.tar.gz && \
```

Comment:
Is this needed for beeline? I thought it just used JDBC/ODBC, so not sure why we need hive installed.

Reply:
`beeline` binary isn't available as standalone I think but it comes with `hive`.